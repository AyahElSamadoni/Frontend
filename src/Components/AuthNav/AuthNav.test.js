import React from 'react';
import { shallow, configure } from 'enzyme';
import authNav from './AuthNav';
import { shallowToJson } from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });
describe('<AuthNav\>', () => {
    it('should be defined', () => {
        expect(<authNav />).toBeDefined();
    });
    it('should render', () => {
        const wrapper = shallow(<authNav />)
        expect(shallowToJson(wrapper)).toMatchSnapshot()
    });
    it('should render', () => {
        const wrapper = shallow(<authNav />)
        expect(wrapper).toMatchSnapshot()
    });
});