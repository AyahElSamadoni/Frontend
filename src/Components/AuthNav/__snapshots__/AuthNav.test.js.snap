// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<authNav> should render 1`] = `<authNav />`;

exports[`<authNav> should render 2`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <authNav />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {},
    "ref": null,
    "rendered": null,
    "type": "authNav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": "authNav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
